common/gossmap.c

```diff
@@ -436,6 +436,13 @@ static struct gossmap_chan *add_channel(struct gossmap *map,
 	return chan;
 }
 
+static u32 peek_cupdate_timestamp(struct gossmap *map, size_t cupdate_off){
+	const size_t timestamp_off = cupdate_off + 2 + 64 + 32 + 8;
```

Comment:
```suggestion
       // what is mean
	const size_t timestamp_off = cupdate_off + 2 + 64 + 32 + 8;
```

We can add a small comment on what these numbers mean? just to simplify the life for people like me that don't know more about this topic

Reply:
Quoted the spec here for clarification.